espree.js

```diff
@@ -5117,6 +5122,10 @@ function parseClassDeclaration() {
     classBody = parseClassBody();
     strict = previousStrict;
 
+    if (expression) {
+        return markerApply(marker, astNodeFactory.createClassExpression(id, superClass, classBody));
```

Comment:
Sounds like the path is to remove this and just allow it to send in a `null` id to `createClassDeclaration` instead....